MaritzSTL/koa-ms
# Feature flags

This lays groundwork for providing the LDClient to every request.

All future clients should be instantiated at/inside/as the result of
1. loading config
2. `ServiceContainer`

**To Test:**
* log out `ctx.config` and see we're getting things from config server

This introduces some work on us to add things to the configuration server.
